lib/cfn-model/transforms/serverless.rb

```diff
@@ -7,10 +7,16 @@ class Transforms
     # https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/transform-aws-serverless.html
     class Serverless
       def perform_transform(cfn_hash)
+        with_line_numbers = false
         resources = cfn_hash['Resources'].clone
         resources.each do |resource_name, resource|
-          next unless resource['Type'].eql? 'AWS::Serverless::Function'
-          replace_serverless_function cfn_hash, resource_name
+          next unless (resource['Type'].is_a?(String) &&
+                       resource['Type'].eql?('AWS::Serverless::Function')) ||
+                      (resource['Type'].is_a?(Hash) &&
```

Comment:
turn this comparison into a predicate to improve readability.  e.g. enriched_with_line_numbers?(resource['Type'])

Reply:
Updated in latest commit.